.travis.yml

```diff
@@ -1,9 +1,22 @@
 language: python
 python:
-  - 2.7
```

Comment:
To be exact, tox create virtualenv according tox.ini and $TOX_ENV since it runs with `-e $TOX_ENV`. But anyway, `*-py27` are in envlist.